src/openCypherParser/AST/CypherVisitor.cs

```diff
@@ -530,6 +530,66 @@ private void UpdateEntityTypesInExpressionInPlace(IEnumerable<QueryExpression> e
             });
         }
 
+        // the order of parsing types referred from following:
```

Comment:
Please use standard comment format (///) or move the comment inside the function